DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -282,6 +284,9 @@ struct globus_l_gfs_iRODS_handle_t
     uint64_t                            irods_parallel_file_size_threshold_bytes;
 
     bool                                first_write_done;
+
+    // added to get file modification time from client
+    time_t                              utime;
```

Comment:
How is `utime` set?
Is it guaranteed to be set to a valid value?

Reply:
The address of it gets sent to the call to `globus_gridftp_server_get_recv_modification_time()`. That function will either set it to what the client requested or will set it to -1 if the client does not request it.  If that function returns an error, I set it to -1.

I only read that after calling `globus_gridftp_server_get_recv_modification_time()` so it will always be set properly.